shared/src/hover/HoverOverlay.tsx

```diff
@@ -170,6 +197,26 @@ export class HoverOverlay extends React.PureComponent<HoverOverlayProps> {
                             })
                     )}
                 </div>
+                {hoverOrError && hoverOrError !== LOADING && !isErrorLike(hoverOrError) && hoverOrError.alerts && (
+                    <div className="hover-overlay__alerts">
+                        {hoverOrError.alerts.map(({ content, type }, i) => (
+                            <div className="hover-overlay__row hover-overlay__alert alert-info" key={i}>
```

Comment:
Since `type` is unique, that should be used as a key. https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318
```suggestion
                            <div className="hover-overlay__row hover-overlay__alert alert-info" key={type}>
```